second-analysis-steps/fixing-errors.md

```diff
@@ -129,89 +129,138 @@ And then run it:
+Before               INFO | +--> /Rich [Address: CLID=0x1 Type=0x2]  (Unloaded) 
+Before               INFO | +--> /Trigger [Address: CLID=0x1 Type=0x2]  DataObject
+Before               INFO | | +--> /RawEvent [Address: CLID=0x3ea Type=0x2]  LHCb::RawEvent
+Before               INFO | +--> /pRec [Address: CLID=0x1 Type=0x2]  DataObject
+Before               INFO | | +--> /Track [Address: CLID=0x1 Type=0x2]  DataObject
+Before               INFO | | | +--> /Best [Address: CLID=0x60e Type=0x2]  LHCb::PackedTracks
+Before               INFO | | | +--> /FittedHLT1VeloTracks [Address: CLID=0x60e Type=0x2]  LHCb::PackedTracks
+Before               INFO | | | +--> /Muon [Address: CLID=0x60e Type=0x2]  (Unloaded) 
+Before               INFO | | +--> /Rich [Address: CLID=0x1 Type=0x2]  DataObject
+Before               INFO | | | +--> /PIDs [Address: CLID=0x619 Type=0x2]  LHCb::PackedRichPIDs
+Before               INFO | | +--> /Muon [Address: CLID=0x1 Type=0x2]  DataObject
+Before               INFO | | | +--> /MuonPID [Address: CLID=0x623 Type=0x2]  LHCb::PackedMuonPIDs
+Before               INFO | | +--> /Calo [Address: CLID=0x1 Type=0x2]  (Unloaded) 
+Before               INFO | | +--> /ProtoP [Address: CLID=0x1 Type=0x2]  DataObject
+Before               INFO | | | +--> /Charged [Address: CLID=0x610 Type=0x2]  LHCb::PackedProtoParticles
+Before               INFO | | | +--> /Neutrals [Address: CLID=0x610 Type=0x2]  (Unloaded) 
+Before               INFO | | +--> /Vertex [Address: CLID=0x1 Type=0x2]  DataObject
+Before               INFO | | | +--> /Primary [Address: CLID=0x611 Type=0x2]  LHCb::PackedRecVertices
+Before               INFO | |   +--> /V0 [Address: CLID=0x612 Type=0x2]  (Unloaded) 
+Before               INFO | +--> /Turbo [Address: CLID=0x1 Type=0x2]  (Unloaded) 
+Before               INFO | +--> /Strip [Address: CLID=0x1 Type=0x2]  (Unloaded) 
+Before               INFO | +--> /AllStreams [Address: CLID=0x1 Type=0x2]  (Unloaded) 
+Before               INFO | +--> /DAQ [No Address]  DataObject
+Before               INFO | | +--> /ODIN [No Address]  LHCb::ODIN
+Before               INFO   +--> /Phys [No Address]  DataObject
+Before               INFO   | +--> /StdAllLooseKaons [No Address]  DataObject
+Before               INFO   | | +--> /Particles [No Address]  KeyedContainer<LHCb::Particle,Co [0x9]
+Before               INFO   | | +--> /decayVertices [No Address]  KeyedContainer<LHCb::Vertex,Cont [0]
+Before               INFO   | | +--> /Particle2VertexRelations [No Address]  LHCb::Relation1D<LHCb::Particle,
+Before               INFO   | | +--> /_RefitPVs [No Address]  KeyedContainer<LHCb::RecVertex,C [0]
+Before               INFO   | +--> /Sel_D0 [No Address]  DataObject
+Before               INFO   | | +--> /Particles [No Address]  KeyedContainer<LHCb::Particle,Co [0]
+Before               INFO     +--> /TupleDstToD0pi_D0ToKpi [No Address]  DataObject
```

Comment:
Should we manually edit this to be `TupleDstToD0pi_D0ToKK`?

Reply:
Well spotted, thanks!